### PR TITLE
Fix filter

### PR DESCRIPTION
**Pull request by aveline**
What this PR does:
- [x] Moves `core.getInput()` to one location
- [x] Renames variables to match the `core.getInput()` name
- [x] Cleans up the message output code
- [x] Fixes new lines missing when not using global install and suffix
- [x] Fixes the filter for `github_comment_included_packages`
- [x] Adds a different default message if the user has `global_install`
- [x] Always renders the default message always and adds the `custom_message_prefix` before it

Tested in https://github.com/Shopify/snapit-cloudsmith-docs/pull/56

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -241,11 +237,15 @@ try {
         .join(',\n') +
       '\n```';
 
+    const defaultMessage = globalInstallMessage
+      ? `Test the snapshot by intalling your package globally:`
+      : `Test the snapshot${multiple ? 's' : ''} by updating your \`package.json\` with the newly published version${multiple ? 's' : ''}:`;
```
Comment (alex-page) [inline index.ts:242]: @isaacroldan I think this is a better developer experience for global packages. I don't think you need to use `customMessagePrefix` for your `snapit` implementation.

File: index.ts [hunk 2]
```diff
@@ -232,7 +228,7 @@ try {
       filteredSnapshots
         .map((pkg) => `${globalInstallMessage} ${pkg.fullString}`)
         .join('\n') +
-      '\n```\n\n';
+      '\n```';
```
Comment (alex-page) [inline index.ts:231]: These new lines were needed for both the global install message or package.json install message. So I moved them to line 248 if there is a `customMessageSuffix`.

File: index.ts [hunk 3]
```diff
@@ -202,25 +204,19 @@ try {
...
-        packageOutputFilter.some((filter) => snapshot.package === filter)
-      );
-    });
+    const filteredSnapshots = githubCommentIncludedPackages
+      ? snapshots.filter((snapshot: Snapshot) =>
+          githubCommentIncludedPackages
+            .split(',')
+            .some((filter) => snapshot.package === filter),
+        )
+      : snapshots;
```
Comment (alex-page) [inline index.ts:213]: The main change is we don't need to filter if there isn't a filter. There was a bug before with the logic for the `!packageOutputFilter ||`

Comment (isaacroldan) [reply index.ts:213]: Yep, sorry about that 🙏

File: index.ts [hunk 4]
```diff
@@ -32,18 +32,20 @@ try {
 
   const buildScript = core.getInput('build_script');
   const isGlobal = core.getInput('global_install') === 'true';
-  const packageOutputFilter = core
-    .getInput('github_comment_included_packages')
-    .split(',');
+  const githubCommentIncludedPackages = core.getInput(
+    'github_comment_included_packages',
```
Comment (isaacroldan) [inline index.ts:36]: is this extra comma needed here? 🤔

**Pull request merged**